### PR TITLE
Feature/issue 693 tests

### PR DESCRIPTION
**Pull request by jcalderin**

### CODE REVIEW

File: test/acceptance/integration/NGSI 10/Delete entity/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+__author__ = 'Jon'
```
Comment (fgalan) [inline test/acceptance/integration/NGSI 10/Delete entity/__init__.py]: License header missing.

File: test/acceptance/integration/NGSI 10/Delete entity/delete.feature [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+# __author__ = 'Jon Calderin Goñi (jon dot caldering at gmail dot com)'
+@delete_entity
+Feature: Subscribe context tests
+
+  Background:
+    Given the Context Broker started with multitenancy
+
+  Scenario: Delete an entity does not exist with convenience and type extended
+    # Append
+    Given a new NGSI version "10" petition with the service "service" and the subservice "/subservice"
```
Comment (fgalan) [inline test/acceptance/integration/NGSI 10/Delete entity/delete.feature]: This probably conflicts with PR #800...

File: test/acceptance/integration/NGSI 10/Subscribe context/__init__.py [hunk 3]
```diff
@@ -0,0 +1 @@
+__author__ = 'Jon'
```
Comment (fgalan) [inline test/acceptance/integration/NGSI 10/Subscribe context/__init__.py]: License header missing.

File: test/acceptance/integration/NGSI 10/Subscribe context/subscribe_context.feature [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+# __author__ = 'Jon Calderin Goñi (jon dot caldering at gmail dot com)'
+
+Feature: Subscribe context tests
+
+  Background:
+    Given the Context Broker started with multitenancy
+  @issue-693
```
Comment (fgalan) [inline test/acceptance/integration/NGSI 10/Subscribe context/subscribe_context.feature]: Issue #693 is a particular case of subscription crash... However, this PR seems to include broad tests about subscription. A bit confusing to me but probably I lack some "context". Let's discuss by skype/F2F.

File: test/acceptance/integration/NGSI 10/__init__.py [hunk 5]
```diff
@@ -0,0 +1 @@
+__author__ = 'Jon'
```
Comment (fgalan) [inline test/acceptance/integration/NGSI 10/__init__.py]: License header missing.

File: test/acceptance/properties.json [hunk 6]
```diff
@@ -17,9 +17,9 @@
   },
   "deploy_data": {
     "host": "localhost",
-    "ssh_port": "22",
-    "user": "root",
-    "password": "root",
-    "bin_path": "/usr/bin/contextBroker"
+    "ssh_port": "",
```
Comment (fgalan) [inline test/acceptance/properties.json]: It's a bit weird having defined properties that you don't use at the end... Should this be mentioned in the README.md for acceptance tests?

Comment (jcalderin) [inline test/acceptance/properties.json]: Fixed in 8dab04b

File: test/acceptance/integration/NGSI 10/__init__.py (hunk 5 above)
Comment (jcalderin) [inline test/acceptance/integration/NGSI 10/__init__.py]: Fixed cbf948b

File: test/acceptance/integration/NGSI 10/Subscribe context/__init__.py (hunk 3 above)
Comment (jcalderin) [inline test/acceptance/integration/NGSI 10/Subscribe context/__init__.py]: Fixed in cbf948b

File: test/acceptance/integration/NGSI 10/Delete entity/__init__.py (hunk 1 above)
Comment (jcalderin) [inline test/acceptance/integration/NGSI 10/Delete entity/__init__.py]: Fixed in cbf948b

File: test/acceptance/integration/NGSI 10/Subscribe context/subscribe_context.feature (hunk 4 above)
Comment (jcalderin) [inline test/acceptance/integration/NGSI 10/Subscribe context/subscribe_context.feature]: Fixed in 8dab04b

File: test/acceptance/integration/NGSI 10/Delete entity/delete.feature (hunk 2 above)
Comment (jcalderin) [inline test/acceptance/integration/NGSI 10/Delete entity/delete.feature]: NTC, it will be checked in https://github.com/telefonicaid/fiware-orion/pull/800

**Comment by fgalan** [comment]
LGTM

File: test/acceptance/integration/terrain.py [hunk 7]
```diff
@@ -57,4 +59,5 @@ def after_each_scenario(scenario):
 @after.all
 def after_all(total):
     drop_all_test_databases(world.config['mongo']['host'], int(world.config['mongo']['port']))
-    stop_cb()
\ No newline at end of file
+    stop_cb()
+    pass
```
Comment (iariasleon) [inline test/acceptance/integration/terrain.py]: this pass is leftover, no?

Comment (jcalderin) [inline test/acceptance/integration/terrain.py]: Fixed in 4e45fc9

**Comment by iariasleon** [comment]
LGTM

**Pull request merged**